acid/templates/builds_history.html

```diff
@@ -32,27 +32,32 @@ <h1>Builds history for {{ buildsets.pipeline }}</h1>
             <div class="col ml-2">Branch</div>
             <div class="col">Build number</div>
             <div class="col">Duration</div>
-            <div class="col-2">Started</div>
-            <div class="col-2">Finished</div>
-            <div class="col-2 text-center">Result</div>
+            <div class="col">Started</div>
+            <div class="col">Finished</div>
+            <div class="col text-center">Result</div>
+            <div class="col text-center" >Permalink</div>
         </div>
         {% for b in buildsets.page %}
-        <div class="row hover-row build-table clickable" data-toggle="collapse"
-             data-target="#collapse-{{ b.id }}">
+        <div class="row hover-row build-table clickable {% if not request.args.unfold %}collapsed{% endif %}"
+             data-toggle="collapse" data-target="#collapse-{{ b.id }}">
             <div class="dropdown-toggle"></div>
             <div class="col">{{ b.branch }}</div>
-            <div class="col"><a class="no-collapsable"
-                                href="{{ buildsets_log_url }}/{{ buildsets.pipeline }}/review.opencontrail.org/{{ b.branch }}/{{ b.build_number }}/"
-                                title="Open logs">{{ show_object_or_na(b.build_number) }}</a></div>
+            <div class="col">
+                <a class="no-collapsable"
+                   href="{{ buildsets_log_url }}/{{ buildsets.pipeline }}/review.opencontrail.org/{{ b.branch }}/{{ b.build_number }}/"
+                   title="Open logs">{{ show_object_or_na(b.build_number) }}</a>
+            </div>
             <div class="col">{{ show_object_or_na(b.duration) }}</div>
-            <div class="col-2">{{ local_datetime(b.start_datetime) }}</div>
-            <div class="col-2">{{ local_datetime(b.end_datetime) }}</div>
+            <div class="col">{{ local_datetime(b.start_datetime) }}</div>
+            <div class="col">{{ local_datetime(b.end_datetime) }}</div>
+            <div class="col text-center">
+                <span class="badge {% if b.result == 'SUCCESS' %} badge-success {% else %} badge-danger {% endif %}">{{ b.result }}</span>
+            </div>
             <div class="col text-center">
-                <span
-                    class="badge {% if b.result == 'SUCCESS' %} badge-success {% else %} badge-danger {% endif %}">{{ b.result }}</span>
+                <a class="btn btn-outline-primary" href="builds?branch={{b.branch}}&build={{b.build_number}}&unfold=true">permalink</a>
```

Comment:
Instead of hardcoding URL, use ``url_for`` like: https://github.com/codilime/acid/blob/2c30d8047b7d9505feba286f1505fdf544e194fe/acid/templates/dashboard.html#L27

Docs: http://flask.pocoo.org/docs/1.0/api/#flask.url_for